### DISCUSSION TITLE
Fade timing issue

### DISCUSSION THREAD

**Discussion by basstwo** [Kounex/obs_station#21] [Q&A]
I have been testing it out.  I have four scenes that I switch between.  In OBS I use Fade at 8000 ms.  In Blade this seems to work for some scene transitions.  But after the transition the Fade time switches from 8000 to 1000 ms.  I can edit it to get back to 8000.  But it doesn't keep it.

Any suggestions?

**Comment by Kounex** [comment]
I thiiiiiiink I found the issue. I pushed version 3.0.2 to the stores which might fix your issue. Let me know once you were able to download the update and test it! :)

**Reply by basstwo** [reply]
OK.  The fade timing seems to persist now.  Thanks!
I did find another bug:  if you are in a slow transition from one scene to another and click on ANOTHER scene mid-transition the button for the third scene _also_ shows that it is transitioning. So, when all is done, it _looks_ like it went to scene three.  But it is still on scene two.

**Comment by basstwo** [comment]
Thanks!  I will check it out!
